xlators/mgmt/glusterd/src/glusterd-volume-set.c

```diff
@@ -1465,6 +1465,14 @@ struct volopt_map_entry glusterd_volopt_map[] = {
      .voltype = "protocol/client",
      .op_version = 1,
      .flags = VOLOPT_FLAG_CLIENT_OPT},
+    {.key = "network.bailout-timeout",
+     .voltype = "protocol/client",
+     .op_version = 1,
+     .flags = VOLOPT_FLAG_CLIENT_OPT},
+    {.key = "network.reconnect-timeout",
+     .voltype = "protocol/client",
+     .op_version = 1,
```

Comment:
can the `.op_version` be `GD_OP_VERSION_10_0` here? (and above).

Reply:
OK